lambdas/db-migration/src/migrations/20201014105054_create_pdrs_table.ts

```diff
@@ -22,8 +19,36 @@ export const up = async (knex: Knex): Promise<void> =>
     table
       .integer('executionCumulusId')
       .references('cumulusId')
-      .inTable('executions')
+      .inTable('executions');
+    table
+      .enum('status', ['running', 'failed', 'completed'])
+      .comment('Status (running, failed, completed) of the PDR')
       .notNullable();
+    table
+      .float('progress')
+      .comment('PDR completion progress percentage');
+    table
+      .boolean('PANSent')
+      .comment('Boolean defining if a PAN response has been sent for this PDR');
+    table
+      .text('PANmessage')
+      .comment('PAN message text');
+    table
+      .jsonb('stats')
+      .comment('PDR stats json object');
+    table
+      .text('address');
+    table
+      .text('originalUrl');
+    table
+      .integer('duration');
+    table
```

Comment:
Super super nit: I kind of like having all the notNullable fields at the top of the migration. But not necessary at all

Reply:
Agreed, and not a nit : 5c44b81ae